### ISSUE TITLE
autocomplete urls with parameter - create does not work

### ISSUE THREAD

**Issue by erdnax123** [yourlabs/django-autocomplete-light#1145]
```
urlpatterns = [
    path('tags/<namespace>/', TagsAutocomplete.as_view(), name='tags-autocomplete')
]`

widget=Model2Select(url=reverse('tags-autocomplete', kwargs={'namespace': 'xyz'})
```
on create new tag

ERROR: post() got an unexpected keyword argument 'namespace'

**Comment by jpic** [comment]
Does it work with `reverse('xyz:tags-autocomplete')`

**Comment by erdnax123** [comment]
in my case namespace is a dynamic value, can also be a project_id ...

in all views the post method is: 
```
def post(request, *args, **kwargs):
```
so i can use in all functions `self.kwargs[...]` my tempory fix in the view is:
```
    def post(self, request, *args, **kwargs):
        self.kwargs = kwargs
        return super().post(request)
```
in get requests `self.kwargs` is correct initialized

see also django.views.generic.View.dispatch

**Comment by jpic** [comment]
Ok so it turns out you're not talking about [url namespaces](https://docs.djangoproject.com/en/3.0/topics/http/urls/#url-namespaces-and-included-urlconfs), but just a url pattern argument called namespace.

So, I've merged your pull request, I think it's released in 2.6.0-dev, can we close this issue ?

**Comment by jpic** [comment]
Thanks for the contrib and keep up the great work :tophat:
